apisix/plugins/limit-count/init.lua

```diff
@@ -68,6 +68,7 @@ local policy_to_additional_properties = {
                 type = "boolean", default = false,
             },
         },
+        encrypt_fields = {"redis_password"},
```

Comment:
I think this is not correct, please refer to https://github.com/apache/apisix/blob/master/apisix/plugins/kafka-proxy.lua#L36, and add a corresponding test case

Reply:
your are correct, that i am doing 2 different things in a single PR, let me ever this change